octoprint_psucontrol/__init__.py

```diff
@@ -545,16 +549,25 @@ def turn_psu_on(self):
             if self.sensingMethod not in ('GPIO','SYSTEM'):
                 self._noSensing_isPSUOn = True
          
-            time.sleep(0.1 + self.postOnDelay)
+            threading.Timer(0.1 + self.postOnDelay, self.after_psu_on_delay).start()
+
 
+    def after_psu_on_delay(self):

```

Comment:
What's the reason for doing this?

Reply:
I've had some strange behavior while using PSU Control. The printing did stall while PSU control was executing external commands to check if the PSU is on. So there has to be some parts blocking Octoprint itself.
I've seen `threading.Timer` in some other plugins and IMHO it is always better to use non-blocking timers wherever it is possible.